python/lsst/ip/diffim/utils.py

```diff
@@ -711,3 +732,354 @@ def plotWhisker(results, newWcs):
+                       height=2.5):
+    """Use matplotlib.pyplot.imshow() to display a afwImage.MaskedImageF,
+    alongside its masks and variance plane
+
+    @see displayImage() for params not listed here
+
+    @param maskedImage MaskedImage to display
+    @param showMasks Display the MaskedImage's masks
+    @param showVariance Display the MaskedImage's variance plane
+
+    @return matplotlib.pyplot.figure dispaying the image
+
+    """
+    plt = importMatplotlib()
+    if not plt:
+        return
+
+    fig = plt.figure(figsize=(width, height))
+    bbox = maskedImage.getBBox()
+    extent = (bbox.getBeginX(), bbox.getEndX(), bbox.getBeginY(), bbox.getEndY())
+    plt.subplot(1, 3, 1)
+    ma = maskedImage.getArrays()
+    display2dArray(ma[0], title='Data', showBars=showBars, extent=extent)
+    if showMasks:
+        plt.subplot(1, 3, 2)
+        display2dArray(ma[1], title='Masks', showBars=showBars, extent=extent)
+    if showVariance:
+        plt.subplot(1, 3, 3)
+        display2dArray(ma[2], title='Variance', showBars=showBars, extent=extent)
+    return fig
+
+
+def displayExposure(exposure, showMasks=True, showVariance=False, showPsf=False, showBars=True,
+                    width=8, height=2.5):
+    """Use matplotlib.pyplot.imshow() to display a afw.image.Exposure,
+    including its masks and variance plane and optionally its Psf.
+
+    @see displayMaskedImage() for params not listed here
+
+    @param exposure Exposure to display
+    @param showPsf Display the exposure's Psf
+
+    @return matplotlib.pyplot.figure dispaying the image
+
+    """
+    plt = importMatplotlib()
+    if not plt:
+        return
+
+    fig = displayMaskedImage(exposure.getMaskedImage(), showMasks,
+                             showVariance=not showPsf,
+                             showBars=showBars, width=width, height=height)
+    if showPsf:
+        plt.subplot(1, 3, 3)
+        psfIm = exposure.getPsf().computeImage()
+        bbox = psfIm.getBBox()
+        extent = (bbox.getBeginX(), bbox.getEndX(), bbox.getBeginY(), bbox.getEndY())
+        display2dArray(psfIm.getArray(), title='PSF', showBars=showBars, extent=extent)
+    return fig
+
+
+def displayCutouts(source, exposure, posImage=None, negImage=None, asHeavyFootprint=False, title=''):
+    """Use matplotlib.pyplot.imshow() to display cutouts within up to
+    three afw.image.Exposure's, given by an input SourceRecord.
+
+    @param source afw.table.SourceRecord defining the footprint to extract and display
+    @param exposure afw.image.Exposure from which to extract the cutout to display
+    @param posImage Second exposure from which to extract the cutout to display
+    @param negImage Third exposure from which to extract the cutout to display
+    @param asHeavyFootprint Display the cutouts as
+    afw.detection.HeavyFootprint, with regions outside the
+    footprint removed
+
+    @return matplotlib.pyplot.figure dispaying the image
+
+    """
+    plt = importMatplotlib()
```

Comment:
An argument could be made for `importMatplotlib()` to throw an exception if matplotlib could not be loaded since you are explicitly calling it because you want to make a plot.